src/components/node-list/index.js

```diff
@@ -175,9 +174,9 @@ const NodeListProvider = ({
 
   // Deselect node on Escape key
   const handleKeyDown = (event) => {
-    utils.handleKeyEvent(event.keyCode, {
-      escape: () => onToggleNodeSelected(null),
-    });
+    if (event.keyCode === 27) {
+      onToggleNodeSelected(null);
+    }
```

Comment:
Nice spot on refactoring this - as suggested, once the `handleKeyEvents` function is set up in utils, we can follow a similar structure as set out before.